spec/controlling-tasks.md

```diff
@@ -147,9 +194,17 @@ To <dfn for="TaskSignal">add a priority change algorithm</dfn> |algorithm| to a
   1. [=Fire an event=] named {{TaskSignal/prioritychange}} at |signal| using
      {{TaskPriorityChangeEvent}}, with its {{TaskPriorityChangeEvent/previousPriority}}
      attribute initialized to |previousPriority|.
+  1. [=list/iterate|For each=] |dependentSignal| of |signal|'s [=TaskSignal/dependent signals=],
+     [=TaskSignal/signal priority change=] on |dependentSignal| with |priority|.
   1. Set |signal|'s [=TaskSignal/priority changing=] to false.
 </div>
 
+### Garbage Collection {#sec-task-signal-garbage-collection}
```

Comment:
Something's off here and is causing the rendered output to include this suffix: https://pr-preview.s3.amazonaws.com/shaseley/scheduling-apis/pull/72.html#sec-task-signal-garbage-collection